clients/include/hipblas_common.yaml

```diff
@@ -2,27 +2,9 @@
 # or enums defined with c_int base clases and attributes.
 Datatypes:
   - int: c_int
-  - hipblasDatatype_t:
+  - hipDataType:
       bases: [ c_int ]
       attr:
```

Comment:
Why don't you have to replicate all the enum values?

Reply:
They aren't all used in the test suite. I'm making another commit though so I'll add them all.